static/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -32,23 +32,29 @@ import {DisplayOption} from '../utils';
 import Header from './header';
 import ProjectLink from './projectLink';
 
-const ADOPTION_STAGE_LABELS: Record<
+export const ADOPTION_STAGE_LABELS: Record<
   string,
   {name: string; tooltipTitle: string; type: keyof Theme['tag']}
 > = {
   low_adoption: {
     name: t('Low Adoption'),
-    tooltipTitle: t('Never exceeded 10% adoption in a 6 hour period'),
+    tooltipTitle: t(
```

Comment:
Now that I am reading these new tooltips I realized they change the meaning of how we calculate adoption. I am not sure they are correct. Can you please verify it with @iProgramStuff and if that's the case then let's just use the old ones or ask @robinrendle to adjust these.